state/protocol/badger/mutator_test.go

```diff
@@ -624,8 +632,16 @@ func TestExtendEpochTransitionValid(t *testing.T) {
 		assert.NoError(t, err)
 		require.Equal(t, flow.EpochPhaseCommitted, phase)
 
+		// expect epoch phase transition once we finalize block 6
+		consumer.On("EpochCommittedPhaseStarted", epoch2Setup.Counter-1, block6.Header)
+		// expect committed final view to be updated, since we are committing epoch 2
+		metrics.On("CommittedEpochFinalView", epoch2Setup.FinalView)
+		metrics.On("CurrentEpochPhase", flow.EpochPhaseCommitted)
 		err = state.Finalize(block6.ID())
 		require.NoError(t, err)
+		consumer.AssertCalled(t, "EpochCommittedPhaseStarted", epoch2Setup.Counter-1, block6.Header)
+		metrics.AssertCalled(t, "CommittedEpochFinalView", epoch2Setup.FinalView)
+		metrics.AssertCalled(t, "CurrentEpochPhase", flow.EpochPhaseCommitted)
```

Comment:
Same note for the transition to `EpochCommitted` phase. We already have the `CommittedEpochFinalView` and `EpochCommittedPhaseStarted` assertions on lines 649 and 647. We only need to add the `CurrentEpochPhase` assertion inline with those (for block 7)